### ISSUE TITLE
[Visual Requirement-Help-Getting Started]: Luminosity ratio is less than 4.5:1 for the text showing in Orange color and Ok button .

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2535]
**User Experience:**
Users who depends on Keyboard with Screen reader will not be able to access the any Textual content if its luminosity ratio is less than 4.5:1.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool Used: Color Contrast Analyzer

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Help control lying in the header section on the page and select it.
4. Navigate to 'Getting started' control and select it.
5. Verify the Luminosity ratio for the all the text showing on orange color using CCA tools.

**Actual Result:-**
Luminosity ratio is less than 4.5:1 for the text showing in Orange color and Ok button on Getting started page.

**Expected Result:-**
Luminosity ratio is less than 4.5:1 for the text showing in Orange color and Ok button on Getting started page.

**Note:** This issue exist throughout app for text showing in Orange color

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested Fix:**
1.1-Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text 
1.2- If design allows , Not specify background color, not specify text color, and not using technology features that change those defaults .In this technique the author avoids having to do any contrast measures by simply not specifying the text color and not specifying the background. As a result the colors of both are completely determined by the user agent.
Tips:-
•The easiest fix is to increase the brightness of light colors and decrease the brightness of darker colors. 
•Exaggerate lightness differences between foreground and background colors, and avoid using colors of similar brightness adjacent to one another, even if they differ in saturation or hue. 
•During design Use the Color Contrast Analyzer to determine whether new colors have color contrast ratio of at least 4.5:1. 
 
**Key References for Fix:** 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
 
**Please refer the attchment:**
![mas1 4 3_help_get started](https://user-images.githubusercontent.com/30459830/28669507-215531a0-72f2-11e7-94ff-c7b3fdb705ea.jpg)

**Comment by v-amik** [comment]
review done

**Comment by veler** [comment]
Fix available on [https://makecode.microbit.org/](https://makecode.microbit.org/)

**Comment by v-mesala** [comment]
Verified on URL:https://makecode.microbit.org/acc and issue is partially fixed. Issue is not fixed for text having Orange background color. Hence reactivating the bug.

**Comment by veler** [comment]
@microsoftsam , does this orange is property of micro:bit too?

**Comment by microsoftsam** [comment]
https://github.com/Microsoft/pxt-microbit/blob/master/theme/site/globals/site.variables#L24

**Comment by microsoftsam** [comment]
Find out where this CSS property is authored, if it links to @orange then you know it's the micro:bit orange. If its a custom orange, then it's not..

**Comment by veler** [comment]
OK, it's micro:bit orange color. Should we pick a new one and submit it to approval before opening a PR?

**Comment by microsoftsam** [comment]
This one doesn't have to use the microbit orange. Use a custom one and make it the same for all targets.

**Comment by v-swasha** [comment]
Verified on https://makecode.microbit.org/, Issue is fixed for ok button and issue is not fixed for "Show string and On start" text. Hence reactivating the bug.
We are not able to Add/Remove labels. Hence not able to add Active label.

**Comment by v-swasha** [comment]
Verified on https://makecode.microbit.org/, Issue is fixed for ok button and issue is not fixed for "Show string and On start" text. Hence reactivating the bug.
I don't have permissions to ADD/Remove labels. Hence not able to add Active label.

**Comment by samelhusseini** [comment]
Issue is now fixed as we now use the colors of blocks for inline blocks. Please verify issue is fixed @v-amik

**Comment by sonimishra** [comment]
Verified on https://makecode.microbit.org/, Issue is fixed for ok button and issue is still not fixed for "Show string and On start" text. Hence reactivating the bug.
I don't have permissions to ADD/Remove labels. Hence not able to add Active label.

**Comment by guillaumejenkins** [comment]
Re-added Active label

**Comment by samelhusseini** [comment]
Please try it on makecode.microbit.org/beta and let me know if the issue is fixed. I tested it and I got a ratio of 4:5:01

**Comment by v-swasha** [comment]
I have verified this on URL:makecode.microbit.org/beta and issue occurs as fixed As i dont have permissions to add tags .So please update the Closed tag as issue is fixed.
